### DISCUSSION TITLE
How to switch to advanced hosting option with Bullhorn

### DISCUSSION THREAD

**Discussion by jvbhosale** [bullhorn/career-portal#484] [Q&A]
We have had the basic hosting option with Bullhorn for a long time and we are planning to switch to advanced hosting option after trying it out. This is mainly so that google caches our job search results as this is for our staffing division that uses Bullhorn for job placements. We followed all the steps for trying out advanced hosting but the site displays job search results with a "#" in the URL which I am told is an indication of basic hosting option. Is the app.json file from Bullhorn that is hosted in the root directory, different for basic and advanced hosting options? Is yes, then what is the difference? Or can we not have the basic and advanced hosting options on the same server? Are there any other considerations that I am missing? 

Thanks in advance for your help.

Jaideep

**Comment by hutchred** [comment]
Hey @jvbhosale were you ever able to get this issue fixed?

**Comment by pbearne** [comment]
If it doesn't workout  you will get better results with matador Jobs check this whitepaper [https://matadorjobs.com/why-replace-oscp-with-matador-jobs/](https://matadorjobs.com/why-replace-oscp-with-matador-jobs/)

**Comment by hutchred** [comment]
Hi, piggy backing off of this discussion as I didnt want to open a new thread on the topic.

@charlesabarnes can you help direct me to the instructions to install the advanced hosting? When we follow this link (https://github.com/bullhorn/career-portal/blob/75c0ddeed143a0afcab6cba2a24f24b1da714cc5/advanced-install.md), it is broken. We've got our server set up and running but it is running the basic site. Any insight to install the advanced would be great as the instructions are unclear and we're not able to populate the sitemap.

Thanks!

**Comment by pbearne** [comment]
Charles no longer works for Bullhorn
so I doubt he will responed

Paul
